packages/kit-forms/src/index.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  CheckboxField,
+  InputField,
+  RadioGroupField,
+  RiseForm,
+  SelectField,
+  SliderField,
+  SubmitButton,
+  SwitchField,
+  TextField,
+  ToggleGroupField,
+} from './form'
+
+export const FormComponents = {
+  '@rise-tools/kit-forms/RiseForm': {
```

Comment:
if we use this deep namespace we can just call this Form, no need for RiseForm

Reply:
True! For the exported constant, I wanted to avoid collision with Tamagui form tho!